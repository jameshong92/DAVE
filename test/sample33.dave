// test fld type conversion
int main() {
	str path = "test.txt";
    tbl test = load(path);
    test = convert(test, 0, "double");
    print(test);
}
