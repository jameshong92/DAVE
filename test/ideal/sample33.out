height	name	gender	weight	;
178.0	tom	male	68	;
195.0	jacky	male	81	;
175.0	jay	male	77	;
60.0	janice	female	48	;
174.0	sammy	female	69	;